src/main/java/com/jcabi/xml/ClasspathSources.java

```diff
@@ -81,16 +86,22 @@ public ClasspathSources(final String pfx) {
     @Override
     public Source resolve(final String href, final String base)
         throws TransformerException {
-        final InputStream stream = this.getClass().getResourceAsStream(
-            String.format("%s%s", this.prefix, href)
+        InputStream stream = this.getClass().getResourceAsStream(
+            String.format(PATTERN, this.prefix, href)
         );
         if (stream == null) {
-            throw new TransformerException(
-                String.format(
-                    "resource \"%s\" not found in classpath with prefix \"%s\"",
-                    href, this.prefix
-                )
+            stream = this.getClass().getResourceAsStream(
+                String.format(PATTERN, base, href)
             );
+            if (stream == null) {
+                throw new TransformerException(
+                        String.format(
+                                "resource \"%s\" not found in classpath"
```

Comment:
@piddubnyi instead of concactenation let's keep it all in 1 line and add `@checkstyle LineLength (1 line)` 1 line above this one.